### PR TITLE
Feature/integral relations

### PR DESCRIPTION
**Pull request by niklasl**
Adds the `:integral` collection and categorizes integral relations. Also adds the new  `:locatedIn` resource (which is defined as `:integral`).

### CODE REVIEW

**Review by olovy — APPROVED** [review]
LGTM!

File: source/vocab/relations.ttl [hunk 1]
```diff
@@ -392,6 +393,7 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     owl:inverseOf :replacedBy .
 
 :hasReproduction a owl:ObjectProperty;
+    :category :integral ; # TODO: is this really the case, or is :reproductionOf enough?
```
Comment (klngwll) [inline source/vocab/relations.ttl:396]: According to the comment on integral (nicely documented by the way 😃 ) it, I would not say a reproduction is necessery to understand the original resource.

**Review by klngwll — APPROVED** [review]
LGTM!

File: source/vocab/relations.ttl (hunk 1 above)
Comment (niklasl) [reply source/vocab/relations.ttl:396]: Probably not (and thanks 😊 ). What I had in mind was the perceived need to show the media of a reproduction alongside a print; but that is not *integral* information (there are lots of interesting things we do show which are not). OTOH it *might* be the case that `:associatedMedia` is `:integral` though, but we'll see.

I'll just remove the `:integral` from `:hasReproduction` for now and we'll see if any such need arises.

Comment (niklasl) [reply source/vocab/relations.ttl]: Removed in  3fd9af2.

File: source/vocab/relations.ttl [hunk 2]
```diff
@@ -393,7 +393,6 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     owl:inverseOf :replacedBy .
 
 :hasReproduction a owl:ObjectProperty;
-    :category :integral ; # TODO: is this really the case, or is :reproductionOf enough?
```
Comment (olovy) [inline source/vocab/relations.ttl]: :+1:

**Pull request merged**